### ISSUE TITLE
Miner using different key

### ISSUE THREAD

**Issue by bentech** [helium/miner#1384]
The docker image will use a generated key if the i2c is misconfigured.

I believe the miner I bought was using a file based ecc key.  There were different public keys for the ```docker exec provision gateway_mfr ecc onboarding``` and ```docker exec miner miner print_keys``` command. The print_keys command would display a different public key when i deleted the swarm_key file .

Once all the config files were deleted both keys matched.

On the discord someone claims their miner changed keys. It appears possible to add the file based key on the blockchain.

**Comment by ke6jjj** [comment]
@bentech most of what you have described is indeed how the software operates. Can you rephrase this in terms of what is _expected_ to happen versus what _actually_ happened?
